themes/app/src/scss/pagetype/_homepage.scss

```diff
@@ -0,0 +1,20 @@
+.homepage__banner-text {
+    background-color: #EDEDED;
```

Comment:
Look to store this hexcode in some sort of color variable i.e. `$color:#EDEDED` so that you can easily modify this color in one place and it will propagate to all other styles that use this variable :)